ci/do_ci.sh

```diff
@@ -593,7 +608,7 @@ case $CI_TARGET in
                 cp -a linux/arm64/envoy "publish/envoy-${version}-linux-aarch_64"
                 cp -a linux/arm64/envoy-contrib "publish/envoy-contrib-${version}-linux-aarch_64"
 
-                "${ENVOY_SRCDIR}ci/publish_github_assets.sh" "v${version}" "${PWD}/publish"
+                "${ENVOY_SRCDIR}/ci/publish_github_assets.sh" "v${version}" "${PWD}/publish"
```

Comment:
Seems this is a fix unrelated to this goal of this PR?

Reply:
yeah, but i think we want to carry it - it seemed reasonable to add here rather than separating to an indiv PR - ill add a message above ...